docs/typography.mdx

```diff
@@ -0,0 +1,156 @@
+---
+name: 'Typography'
+---
+
+<!-- workaround to drop swingset's styling of first h1 -->
+
+<h1 />
```

Comment:
Copied this pattern from the `color.mdx` page as I had the same intent for the `h1`.